integration-tests/keycloak-authorization/src/main/java/io/quarkus/it/keycloak/AdminClientResource.java

```diff
@@ -29,22 +29,22 @@ public class AdminClientResource {
     Keycloak keycloak;
 
     @GET
-    @Produces(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.TEXT_PLAIN)
     @Path("realm")
-    public RealmRepresentation getRealm() {
-        return keycloak.realm("quarkus").toRepresentation();
+    public String getRealm() {
+        return keycloak.realm("quarkus").toRepresentation().getRealm();
     }
 
     @GET
-    @Produces(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.TEXT_PLAIN)
     @Path("newrealm")
-    public RealmRepresentation createRealm() {
+    public String createRealm() {
```

Comment:
Why is `ReamRepresentation` is replaced with `String`  here ?

Reply:
Because the `ObjectMapperCustomizer` I added customizes the server response, which means that you can no longer properly serialize it and expect the test code to be able to deserialize.

Reply:
@geoand Oh yes, I see, KeycloakAdminClient itself returns a correct instance